### PR TITLE
doc: add README logo

### PR DESCRIPTION
**Pull request by syntaxval**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# otrails
+<img width="200" alt="Maputnik logo" src="./doc/otrails.svg" />
```
Comment (drmats) [inline README.md]: Please change 'alt' attribute.

**Review by drmats — CHANGES_REQUESTED** [review]
Just one change. Otherwise - awesome!

File: README.md (hunk 1 above)
Comment (syntaxval) [reply README.md]: Lol

**Review by drmats — APPROVED** [review]
:-)

**Pull request merged**